.github/scripts/doxygen.sh

```diff
@@ -8,6 +8,18 @@ set -o nounset # Exit if variable not set.
 set -o pipefail # Exit if pipe failed.
 
 sudo apt update
-sudo apt install -y doxygen
+sudo apt install -y git cmake python flex bison
+
+pushd ~
+git clone -b Release_1_9_0 https://github.com/doxygen/doxygen.git
```

Comment:
do we know which feature we are using that is causing us to pin to this version ? If we can remove then we can avoid this building of things.

Reply:
Yea check the link above in the PR. 1.8.17 introduced a regression in that feature (which is crucial) and we therefore are jumping to the next major release. Unfortunately that regression is the same version in apt-get for Ubuntu 20.04. I confirmed at least locally that it works